src/Avalonia.FreeDesktop/Avalonia.FreeDesktop.csproj

```diff
@@ -12,8 +12,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Tmds.DBus.Protocol" Version="0.14.0" />
-    <PackageReference Include="Tmds.DBus.SourceGenerator" Version="0.0.5" />
+    <PackageReference Include="Tmds.DBus.Protocol" Version="0.14.0" PrivateAssets="All" />
```

Comment:
@kekekeks I also marked `Tmds.DBus.Protocol` with `PrivateAssets="All"`. If that is incorrect please let me know and I'll fix it.

Reply:
Nevermind: Had to revert this part because it didn't build. See comment below.